class-two-factor-core.php

```diff
@@ -480,6 +483,23 @@ public static function filter_authenticate( $user ) {
 		return $user;
 	}
 
+	/**
+	 * Prevent login cookies being set on login for Two Factor users.
+	 *
```

Comment:
```suggestion
	 *
	 * This makes it so that Core never sends the auth cookies. `login_form_validate_2fa()` will send them manually once the 2nd factor has been verified.
	 *
```